ui/app/manager/src/pages/page-alarms.ts

```diff
@@ -0,0 +1,1133 @@
+
+                if (e.response.status === 400) {
+                    showSnackbar(
+                        undefined,
+                        i18next.t(isUpdate ? "alarm.saveAlarmFailed" : "alarm.createAlarmFailed"),
+                        i18next.t("dismiss")
+                    );
+                } else if (e.response.status === 403) {
+                    showSnackbar(undefined, i18next.t("alarm.alarmAlreadyExists"));
+                }
+            }
+            throw e; // Throw exception anyhow to handle individual cases
+        } finally {
+            await this._loadData();
+            this.reset();
+        }
+    }
+
+
+
+    protected render() {
+        if (!manager.authenticated) {
+            return html`
+                <or-translate value="notAuthenticated"></or-translate> `;
+        }
+
+        const disabled = false;
+        const readAlarms = manager.hasRole("read:alarms");
+        const writeAlarms = manager.hasRole("write:alarms");
+
+        const readonly = readAlarms && !writeAlarms;
+        const assignOnly = !readAlarms && !writeAlarms;
+        this.config = {assignOnly: assignOnly};
+        return html`
+            <div id="wrapper">
+                <!-- Alarm Specific page -->
+
+                <div id="title" class="${this.creationState || this.alarm ? "hidden" : ""}" style="justify-content: space-between; margin-top: 10px; margin-bottom: 10px">
+                    <div class="${this.creationState || this.alarm ? "hidden" : ""}">
+                        <or-icon icon="bell-outline" style="margin-left: 20px"></or-icon>
+                        <span> ${this.alarm != undefined ? this.alarm.title : i18next.t("alarm.alarm_plural")} </span>
+                    </div>
+                    <div>
+                        <div id="controls">
+                            <div class="${this.creationState || this.alarm || assignOnly || readonly ? "hidden" : "panel-title"}"
+                                 style="justify-content: flex-end;">
+                                <or-mwc-input
+                                        outlined
+                                        class="${this._selected?.length > 0 ? "" : "hidden"}"
+                                        style="margin: 0;"
+                                        type="${InputType.BUTTON}"
+                                        icon="delete"
+                                        @or-mwc-input-changed="${() => (this._deleteAlarms())}"
+                                ></or-mwc-input>
+                            </div>
+                            <hr class="${this._selected?.length > 0 ? "" : "hidden"}">
+                            <div class="${this.creationState || this.alarm ? "hidden" : "controls-left"}">
+                                <or-mwc-input .type="${InputType.CHECKBOX}" id="assign-check"
+                                              ?disabled="${disabled}" .label="${i18next.t("alarm.assignedToMe")}"
```

Comment:
With recent updates you don't have to do `i18next.t()` for the labels anymore, they're automatically translated now by or-mwc-input. (please double check)

Reply:
That does not seem to work.